ownCloud/Client/SortBar.swift

```diff
@@ -240,10 +245,16 @@ class SortBar: UIView, Themeable, UIPopoverPresentationControllerDelegate {
 	// MARK: - Actions
 	@objc private func presentSortButtonOptions(_ sender : UIButton) {
 		let tableViewController = SortMethodTableViewController()
-		tableViewController.modalPresentationStyle = UIModalPresentationStyle.popover
+		tableViewController.modalPresentationStyle = .popover
 		tableViewController.sortBarDelegate = self.delegate
 		tableViewController.sortBar = self
 
+		if #available(iOS 13, *) {
+			tableViewController.tableView.contentInsetAdjustmentBehavior = .never
+			tableViewController.tableView.contentInset = UIEdgeInsets(top: 13, left: 0, bottom: 0, right: 0)
```

Comment:
@felix-schwarz move margin constant into separate definition?

Reply:
Added a local constant and added a comment to describe what this fixes.